### PR TITLE
feat(tests): fuzzy slot migration test

### PR DESCRIPTION
**Pull request by dranikpg**
MVP PR

### CODE REVIEW

File: tests/dragonfly/cluster_test.py [hunk 1]
```diff
@@ -837,3 +837,131 @@ async def test_cluster_slot_migration(df_local_factory: DflyInstanceFactory):
...
+                continue
+
+            print(node, "migrates to", dest_idx, "slots", dest_slots)
+            await nodes[dest_idx].admin_client.execute_command(
+                "DFLYCLUSTER",
+                "START-SLOT-MIGRATION",
+                "127.0.0.1",
+                node.instance.admin_port,
+                *itertools.chain(*dest_slots),
+            )
```
Comment (chakaz) [inline tests/dragonfly/cluster_test.py:928]: Add assert that we received OK?

File: tests/dragonfly/cluster_test.py [hunk 2]
```diff
@@ -837,3 +837,131 @@ async def test_cluster_slot_migration(df_local_factory: DflyInstanceFactory):
...
+        else:
+            break
+
+        iterations += 1
+        assert iterations < 100
+
+        await asyncio.sleep(0.1)
+
+    # Stop seeder
+    seeder.stop()
```
Comment (chakaz) [inline tests/dragonfly/cluster_test.py:952]: Maybe give it a few more seconds to run after stable sync is done, before stopping it?

**Review by chakaz — COMMENTED** [review]
Nicely done, Vlad! And it's cool how many shortcuts Python allows :)

File: tests/dragonfly/cluster_test.py (hunk 1 above)
Comment (dranikpg) [reply tests/dragonfly/cluster_test.py:1040]: Collecting the sync id now

File: tests/dragonfly/cluster_test.py (hunk 2 above)
Comment (dranikpg) [reply tests/dragonfly/cluster_test.py:1068]: Added

**Comment by chakaz** [comment]
Closing this pull request because I'm trying to take over and something does not work correctly because @dranikpg is using a fork

**Pull request closed without merging**